frontend/src/App.js

```diff
@@ -33,73 +46,75 @@ const LoginContainer = () => (
+                  <NavItem eventKey="clientform" active={location.pathname === '/clientform'} >
+                    <NavIcon>
                       <FontAwesomeIcon icon={faUser} />
-                      <span style={{ fontSize: '10px', position: 'absolute', top: '15px', left: '17px' }}>Profile</span>
+                      <span style={{ fontSize: '0.625rem', position: 'absolute', top: '0.9375rem', left: '1.0625rem' }}>Profile</span>
                     </NavIcon>
 
                   </NavItem>
 
-                  <NavItem eventKey="clientplan" >
+                  <NavItem eventKey="clientplan" active={location.pathname === '/clientplan'}>
                     <NavIcon>
                       <FontAwesomeIcon icon={faChartBar} />
-                      <span style={{ fontSize: '10px', position: 'absolute', top: '15px', left: '22px' }}>Plan</span>
+                      <span style={{ fontSize: '0.625rem', position: 'absolute', top: '0.9375rem', left: '1.375rem' }}>Plan</span>
                     </NavIcon>
                   </NavItem>
 
 
-                  <NavItem eventKey="clientactionitems">
+                  <NavItem eventKey="clientactionitems" active={location.pathname === '/clientactionitems'} >
                     <NavIcon>
                       <FontAwesomeIcon icon={faCheckCircle} />
-                      <span style={{ fontSize: '10px', position: 'absolute', top: '15px', left: '4px' }}>Action Items</span>
+                      <span style={{ fontSize: '0.625rem', position: 'absolute', top: '0.9375rem', left: '0.25rem' }}>Action Items</span>
                     </NavIcon>
                   </NavItem>
 
-                  <NavItem eventKey="clientadvisorcontact">
+                  <NavItem eventKey="clientadvisorcontact" active={location.pathname === '/clientadvisorcontact'}>
                     <NavIcon>
                       <FontAwesomeIcon icon={faAddressCard} />
-                      <span style={{ fontSize: '10px', position: 'absolute', top: '15px', left: '4px' }}>Advisor Contact</span>
+                      <span style={{ fontSize: '0.625rem', position: 'absolute', top: '0.9375rem', left: '0.25rem' }}>Advisor Contact</span>
                     </NavIcon>
                   </NavItem>
 
-                  <div style={{ paddingBottom: '40vh' }}></div>
+                  <div style={paddingBottom}></div>
 
-                  {/*TODO*/}
-                  <NavItem eventKey="logout">
+  
+                  <NavItem eventKey="/" onClick={() => window.location.reload(history.push("/"))} >
                     <NavIcon>
                       <FontAwesomeIcon icon={faPowerOff} />
-                      <span style={{ fontSize: '10px', position: 'absolute', top: '15px', left: '17px' }}>Logout</span>
-                    </NavIcon>
-                  </NavItem>
-
+                      <span style={{ fontSize: '0.625rem', position: 'absolute', top: '0.9375rem', left: '1.0625rem' }}>Logout</span>
```

Comment:
this should be a const css at the top of this file but nice job on the px -> rem